flytekit/__init__.py

```diff
@@ -1,5 +1,20 @@
-from __future__ import absolute_import
+import logging as _logging
 
 import flytekit.plugins  # noqa: F401
 
-__version__ = "0.12.6"
+__version__ = "0.13.0b2"
+
+logger = _logging.getLogger("flytekit")
+
+# create console handler and set level to debug
+ch = _logging.StreamHandler()
+ch.setLevel(_logging.DEBUG)
```

Comment:
do you want the level to be DEBUG?

Reply:
Yup.  At least for the beta releases right?  I'd like to add a lot more debug statements and some info statements too.  I'll make it info for the non-beta release.